docs/streams/developer-guide/security.html

```diff
@@ -67,6 +67,14 @@
             <p>When applications are run against a secured Kafka cluster, the principal running the application must have the ACL
                 <code class="docutils literal"><span class="pre">--cluster</span> <span class="pre">--operation</span> <span class="pre">Create</span></code> set so that the application has the permissions to create
                 <a class="reference internal" href="manage-topics.html#streams-developer-guide-topics-internal"><span class="std std-ref">internal topics</span></a>.</p>
+	    <p>If you don't want to provide this permission to your application you can create the required internal topics manually.
```

Comment:
suggest: `To avoid providing this permission to your application, you can create the required internal topics manually. If the internal topics exist, Kafka Streams will not try to recreate them.`